frontend/src/concepts/userSSAR/useAccessAllowed.ts

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { AccessReviewResourceAttributes } from '~/k8sTypes';
+import { AccessReviewContext } from '~/concepts/userSSAR/AccessReviewContext';
+
+/**
+ * Check if the user has access to the provided resource attributes.
+ * Consider using one of the other options.
+ *   - <AccessAllowed /> for simple rendering blocks
+ *   - useKebabAccessAllowed() for kebab options
+ * If you still want to use this directly, consider using verbModelAccess for resourceAttributes.
+ * @see AccessAllowed
+ * @see useKebabAccessAllowed
+ * @see verbModelAccess
+ */
+export const useAccessAllowed = (
```

Comment:
This function is going to get mixed up with the `useAccessReview` hook if we leave them as-is... maybe we name all these items in this folder `...CachedAccessAllowed`?

Reply:
so would the expectation be to use this hook instead of useAccessReview if you just need to simply check access?

Reply:
Yeah, well -- that's the question I'm stuck on. Do we have a "cached check and a not cached check" or do we do everything under the cache -- will that cause problems if you want to check your users access to something live -- instead of having a cached value...

Admins having it, makes sense... but honestly this could extend to _all access checks_ if we can decide if it's a good idea to cache everything.

Reply:
Granting permissions or revoking permissions is a wide guess... I'm not sure. 

If I reduced my permissions on the project page to a theoretical `view` state (we don't have this today) -- the UI won't update, even if the calls are remade because of the cache.